compiler/parser/parser.peg

```diff
@@ -443,7 +443,8 @@ SampleProc
           "keys": ARRAY(MAP("kind": "Assignment",
                    "lhs": MAP("kind": "ID", "name": "shape"),
                    "rhs": MAP("kind": "Call", "name": "typeof",
-                              "args": ARRAY(e)))),
+                              "args": ARRAY(e),
+                              "where": NULL))),
```

Comment:
Should we add `"where": NULL` to all of the `"kind": "Call"` maps? (Honest question because I forget what our philosophy is here regarding NULL map values.)

Reply:
Good point let me double check, though I'm pretty sure I wouldn't have added this if this wasn't causing something to fail.

Reply:
This was needed because this was causing our ztests to fail that was comparing the raw results of pigeon to the raw results of ast.Proc. I fixed this by having ast.Proc omit the where clause in ast.Call if where is empty. I think this is a better route.